### PR TITLE
create models and admin

### PR DESCRIPTION
**Pull request by tach-by**
Task. 26
Your task is to create models and configure an admin panel to manage projects and tasks. The Project model should be a project with the name, description, and creation date fields. The Task model must be a project-specific task with the fields of name, description, status and priority. 
The Admin Panel should provide convenient management of projects and tasks, including creating, editing, and deleting records.

### CODE REVIEW

File: apps/project/models.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class Project(models.Model):
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    creation_date = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+class Task(models.Model):
+    STATUS_CHOICES = [
```
Comment (pareikoVladislav) [inline apps/project/models.py]: Лучше определяй подобные константы вне модели. В модели прописываем конкретно поля.

File: apps/project/models.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        ('TO_DO', 'To Do'),
+        ('IN_PROGRESS', 'In Progress'),
+        ('DONE', 'Done'),
+    ]
+
+    PRIORITY_CHOICES = [
```
Comment (pareikoVladislav) [inline apps/project/models.py]: так же.

**Review by pareikoVladislav — CHANGES_REQUESTED** [review]

File: apps/project/models.py (hunk 1 above)
Comment (tach-by) [reply apps/project/models.py]: done

File: apps/project/models.py (hunk 2 above)
Comment (tach-by) [reply apps/project/models.py]: done

**Review by pareikoVladislav — APPROVED** [review]

**Pull request merged**